README.md

```diff
@@ -274,16 +279,21 @@ The contents of this repository is licensed under the
 Your use of Firebase is governed by the
 [Terms of Service for Firebase Services](https://firebase.google.com/terms/).
 
+[gh-abtesting]: https://github.com/firebase/firebase-ios-sdk/abtesting
```

Comment:
Should be below the next line and end with `badge.svg`.

Check the preview by clicking the document icon in the upper right corner of the code review screen.